pyccl/cosmology.py

```diff
@@ -58,7 +62,9 @@ class MatterPowerSpectra(Enum):
     'linear': lib.linear,
     'emu': lib.emu,
     'calculator': lib.pknl_from_input,
-    'camb': lib.pknl_from_boltzman
+    'camb': lib.pknl_from_boltzman,
+    'baccoemu_nonlinear' : lib.baccoemu_nonlinear,
+    'ee2_nonlinear' : lib.ee2_nonlinear
```

Comment:
We may start including lots of other emulators. Why not put them all under a single `'emulator'` label?

Reply:
Agree, now there is a unified emulator label